prisma/schema/application.prisma

```diff
@@ -19,29 +14,52 @@ enum NextAction {
   CANDIDATE_INFORMED
 }
 
+enum QualificationType16 {
+  GCSE
+  INDIAN_X
+  MALAYSIAN
+  SINGAPORE
+  OTHER
+}
+
+enum QualificationType18 {
+  A_LEVEL
+  INDIAN_XII
+  MALAYSIAN
+  SINGAPORE
+  ROMANIAN
+  OTHER
+}
+
 model Application {
-  id                    Int             @id @default(autoincrement())
-  applicant             Applicant       @relation(fields: [applicantId], references: [id])
+  id                    Int                  @id @default(autoincrement())
+  applicant             Applicant            @relation(fields: [applicantId], references: [id])
   applicantId           Int
   admissionsCycle       Int
   applicationDate       DateTime
   wideningParticipation Boolean
   hasDisability         Boolean
   feeStatus             FeeStatus
-  tmuaPaper1Score       Decimal         @db.Decimal(2, 1)
-  tmuaPaper2Score       Decimal         @db.Decimal(2, 1)
-  tmuaOverallScore      Decimal         @db.Decimal(2, 1)
-  qualifications        Qualification[]
-  nextAction            NextAction      @default(TMUA_CANDIDATE)
-  reviewer              User?           @relation(fields: [reviewerId], references: [id])
+  tmuaPaper1Score       Decimal?             @db.Decimal(2, 1)
+  tmuaPaper2Score       Decimal?             @db.Decimal(2, 1)
+  tmuaOverallScore      Decimal?             @db.Decimal(2, 1)
+  age16ExamType         QualificationType16?
+  age16Score            Decimal?             @db.Decimal(3, 1)
+  age18ExamType         QualificationType18?
+  age18Score            Decimal?             @db.Decimal(3, 1)
+  imperialReview        ImperialReview?
+  nextAction            NextAction           @default(TMUA_CANDIDATE)
+  reviewer              User?                @relation(fields: [reviewerId], references: [id])
   reviewerId            Int?
 }
 
-model Qualification {
```

Comment:
Did we decide to drop the notion of qualification and revert to a super table, then?

Reply:
Yes, reason being the A-level equivalent and GCSE-equivalent qualifications each had different enums for the type of exam.